### ISSUE TITLE
 dataset for segmenting marine animals be placed?

### ISSUE THREAD

**Issue by Lei0829** [WZH0120/SAM2-UNet#32]
Hello, how should the dataset for segmenting marine animals be placed?

**Comment by xiongxyowo** [comment]
Hi, following the existing settings, MAS3K and RMAS are not trained together. In other words, we train one model on the training set of MAS3K and tested it on the test set of MAS3K, while also training another model on RMAS.

**Comment by Lei0829** [comment]
> Hi, following the existing settings, MAS3K and RMAS are not trained together. In other words, we train one model on the training set of MAS3K and tested it on the test set of MAS3K, while also training another model on RMAS.

我使用MAS3K的数据集，测试出来的效果和原先图片标注的效果差距很大
![image](https://github.com/user-attachments/assets/b0f698bf-f789-4a0f-96a6-93c6970ab348)
![image](https://github.com/user-attachments/assets/cf36dbf6-278e-4481-8038-41224f6106d1)

**Comment by xiongxyowo** [comment]
Hi, since the network is trained at 352 resolution, it is normal for predictions to look blurrier than GT in some cases. You can also check our pre-computed maps in google drive.

**Comment by Lei0829** [comment]
> Hi, since the network is trained at 352 resolution, it is normal for predictions to look blurrier than GT in some cases. You can also check our pre-computed maps in google drive.

会不会是我训练轮数按你给我的20次过少，改成200--epoch 200 \
--lr 0.001 \
--batch_size 12
还需要改动什么么

**Comment by Lei0829** [comment]
> Hi, since the network is trained at 352 resolution, it is normal for predictions to look blurrier than GT in some cases. You can also check our pre-computed maps in google drive.

我没有明白你说的GT是什么意思

**Comment by xiongxyowo** [comment]
Hi, SAM2-UNet generally converges within 20 epochs. Thus, increasing the number of epochs might not be helpful. Although the example you present seems not good, SAM2-UNet still attains state-of-the-art performance in the task of marine animal segmentation. You can design more advanced segmentation decoders to obtain better segmentation results.

**Comment by Lei0829** [comment]
> Hi, since the network is trained at 352 resolution, it is normal for predictions to look blurrier than GT in some cases. You can also check our pre-computed maps in google drive.

我可以直接在train.py里提高分辨率么？有什么比较好的分辨率推荐

**Comment by Lei0829** [comment]
> Hi, SAM2-UNet generally converges within 20 epochs. Thus, increasing the number of epochs might not be helpful. Although the example you present seems not good, SAM2-UNet still attains state-of-the-art performance in the task of marine animal segmentation. You can design more advanced segmentation decoders to obtain better segmentation results.

![d1eeb5cd255b9ffad5dfc4d94e7775a3_](https://github.com/user-attachments/assets/0b1e0854-56ca-4c9a-9a40-fc047b3bb300)
当我在尝试使用200epoch时候，两个指标提高了两个点，这是过拟合了？
